ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -14,28 +14,27 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.airbnb.lottie.compose.LottieAnimation
 import com.airbnb.lottie.compose.LottieCompositionSpec
 import com.airbnb.lottie.compose.LottieConstants
 import com.airbnb.lottie.compose.animateLottieCompositionAsState
 import com.airbnb.lottie.compose.rememberLottieComposition
 import org.cru.godtools.base.ui.theme.GodToolsTheme
-import org.cru.godtools.tutorial.R
 
-@Preview(showBackground = true)
 @Composable
-internal fun TipsTutorialLayout() = GodToolsTheme() {
+internal fun TipsTutorialLayout(anim: Int, title: Int, body: Int, body2: Int = 0) = GodToolsTheme() {
```

Comment:
the one other thing I would change about the `body2` param is to make it nullable (and default to null) if you aren't displaying a body2 message.